### PR TITLE
Textures for mercenary things

### PR DESCRIPTION
**Pull request by ErhardSteinhauer**
## About the PR
Made textures for a bunch of mercenary things:
- Locker and Airlocks (Standard and Glass);
- PDA, ID, Job Icon;
- Duffel, Satchel, Jumpskirt, Headset;
- Hardsuit with Headpiece, Magboots.

But I don't know how to make actual items, so this PR contains textures only.

## Why / Balance
Uhh..

## Media
Locker:
mercenary mercenary_door mercenary_open

Airlocks:
opening
opening

Icons for items (showing only icons so not to clutter PR too much):
icon icon icon icon
icon icon icon-on pda-mercenary

- [X] I have added screenshots/videos to this PR showcasing its changes

**Changelog**
:cl: erhardsteinhauer
- add: Textures for Mercenary related objects: locker, airlocks, missing clothing (duffel, satchel, jumpskirt), PDA/ID/Job Icon, headset, hardsuit with headpiece.

### CODE REVIEW

File: Resources/Textures/_NF/Clothing/Back/Duffels/mercenary.rsi/merc-inhand-left.xcf
Comment (Cheackraze) [inline Resources/Textures/_NF/Clothing/Back/Duffels/mercenary.rsi/merc-inhand-left.xcf:1]: this snuck in here, should remove

**Review by Cheackraze — CHANGES_REQUESTED** [review]
other than the extra file, it seems like you've forgotten to add meta.json files to some of these texture folders

**Comment by Cheackraze** [comment]
the RSI validator says some of your meta files are incorrect still on these

**Comment by ErhardSteinhauer** [comment]
Alright, I think I fixed issues with meta.json files flagged by RSI validator

File: Resources/Textures/_NF/Clothing/Back/Duffels/mercenary.rsi/merc-inhand-left.xcf
Comment (ErhardSteinhauer) [reply Resources/Textures/_NF/Clothing/Back/Duffels/mercenary.rsi/merc-inhand-left.xcf:1]: deleted

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**